doc/read-the-docs-site/background/credentials.rst

```diff
@@ -93,14 +105,17 @@ Script credentials offer many benefits over public key credentials:
 * Via the datum, they can be associated with supplemental information, which
   can be used for identity verification.
 * They can have failsafe options built into them to allow recovery of the credential
-  if auxillary data such as private keys are compromised.
+  if auxillary data, such as private keys, are compromised.
```

Comment:
Suggest slight modifications for improved clarity.

Script credentials offer many benefits over public key credentials:

* They can encode sophisticated security requirements, such as requiring multiple signatures, limiting asset withdrawal amounts, and enforcing time limits.
* They can be associated with supplemental information via the datum, which can be used for identity verification.
* They can include failsafe options to allow credential recovery if auxiliary data, such as private keys, are compromised.